### ISSUE TITLE
JS Error - Use of Mutation Events is deprecated

### ISSUE THREAD

**Issue by abby999** [paypal/paypal-checkout-components#1067]
getting the following JS error in Production - 

Use of Mutation Events is deprecated. Use MutationObserver instead. script.js:60:8
Blocked loading mixed active content “http://pp.netparcel.com/myaccountinfo.action”[Learn More] viewpaymentinfo.action
ppxo_xc_ppbutton_error 
Object { error: "./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.RESPONSE]@https://www.paypalobjects.com/api/checkout.js:4570:31\nreceiveMessage@https://www.paypalobjects.com/api/checkout.js:4612:60\nmessageListener@https://www.paypalobjects.com/api/checkout.js:4633:13\n", timestamp: 1553452938306, windowID: "fcf5ca96ad", pageID: "f875db692a", prev_corr_ids: "", referer: "https://pp.netparcel.com", host: "www.paypal.com", path: "/webapps/hermes/button", env: "production", country: "CA", … }
 

 ./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.RESPONSE]@https://www.paypalobjects.com/api/checkout.js:4570:31
receiveMessage@https://www.paypalobjects.com/api/checkout.js:4612:60
messageListener@https://www.paypalobjects.com/api/checkout.js:4633:13
logger.js:63:12
ppxo_unhandled_error 
Object { stack: "An error was encountered while handling error:\n\n ./node_modules/zoid/src/component/parent/index.js/_class</ParentComponent.prototype.listeners/_ref9[constants.POST_MESSAGE.ERROR]@https://www.paypalobjects.com/api/checkout.js:7529:32\n_loop/listener<@https://www.paypalobjects.com/api/checkout.js:5775:32\n./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.REQUEST]/<@https://www.paypalobjects.com/api/checkout.js:4540:24\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.try@https://www.paypalobjects.com/api/checkout.js:5660:30\n./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.REQUEST]@https://www.paypalobjects.com/api/checkout.js:4536:17\nreceiveMessage@https://www.paypalobjects.com/api/checkout.js:4612:60\nmessageListener@https://www.paypalobjects.com/api/checkout.js:4633:13\n\n\nonError@https://pp.netparcel.com/admin/viewpaymentinfo.action:473:23\npromisify/</<@https://www.paypalobjects.com/api/checkout.js:9305:28\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.try@https://www.paypalobjects.com/api/checkout.js:5660:30\npromisify/<@https://www.paypalobjects.com/api/checkout.js:9304:24\nresultValue@https://www.paypalobjects.com/api/checkout.js:6916:32\nonce/<@https://www.paypalobjects.com/api/checkout.js:8717:33\ndecorate/</<@https://www.paypalobjects.com/api/checkout.js:14026:66\n_loop@https://www.paypalobjects.com/api/checkout.js:5529:42\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.dispatch@https://www.paypalobjects.com/api/checkout.js:5543:57\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.reject@https://www.paypalobjects.com/api/checkout.js:5503:17\n_loop/<@https://www.paypalobjects.com/api/checkout.js:5541:29\n_loop@https://www.paypalobjects.com/api/checkout.js:5529:42\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.dispatch@https://www.paypalobjects.com/api/checkout.js:5543:57\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.reject@https://www.paypalobjects.com/api/checkout.js:5503:17\n_loop@https://www.paypalobjects.com/api/checkout.js:5525:33\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.dispatch@https://www.paypalobjects.com/api/checkout.js:5543:57\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.reject@https://www.paypalobjects.com/api/checkout.js:5503:17\nZalgoPromise/<@https://www.paypalobjects.com/api/checkout.js:5458:42\nrequest/</<@https://www.paypalobjects.com/api/checkout.js:16160:21\n\n./node_modules/zoid/src/component/parent/index.js/_class</ParentComponent.prototype.error/<@https://www.paypalobjects.com/api/checkout.js:7781:27\n_loop@https://www.paypalobjects.com/api/checkout.js:5529:42\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.dispatch@https://www.paypalobjects.com/api/checkout.js:5543:57\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.then@https://www.paypalobjects.com/api/checkout.js:5559:17\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.prototype.catch@https://www.paypalobjects.com/api/checkout.js:5563:24\n./node_modules/zoid/src/component/parent/index.js/_class</ParentComponent.prototype.error@https://www.paypalobjects.com/api/checkout.js:7780:20\n./node_modules/zoid/src/component/parent/index.js/_class</ParentComponent.prototype.listeners/_ref9[constants.POST_MESSAGE.ERROR]@https://www.paypalobjects.com/api/checkout.js:7529:21\n_loop/listener<@https://www.paypalobjects.com/api/checkout.js:5775:32\n./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.REQUEST]/<@https://www.paypalobjects.com/api/checkout.js:4540:24\n./node_modules/zalgo-promise/src/index.js/promise_ZalgoPromise</ZalgoPromise.try@https://www.paypalobjects.com/api/checkout.js:5660:30\n./node_modules/post-robot/src/index.js/_RECEIVE_MESSAGE_TYPE[conf.b.POST_MESSAGE_TYPE.REQUEST]@https://www.paypalobjects.com/api/checkout.js:4536:17\nreceiveMessage@https://www.paypalobjects.com/api/checkout.js:4612:60\nmessageListener@https://www.paypalobjects.com/api/checkout.js:4633:13\n", errtype: "[object Error]", timestamp: 1553452938481, windowID: "bae32d2c24", pageID: "27911f8e29", prev_corr_ids: "", referer: "pp.netparcel.com", host: "pp.netparcel.com", path: "/admin/viewpaymentinfo.action", env: "production", … }
logger.js:63:12

JS Code - 

   <!-- -------------------------- PayPal Button ------------- Start -->

                                                                                                                <div id="paypal-button-container"></div>

                                                                                                                <!-- 

                                                                                                                <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>

                                                                                                                -->

                                                                                                                                                                <script src="https://www.paypalobjects.com/api/checkout.js"></script>

                                                                                                    <script>

                                                                                                                // env: 'sandbox', // sandbox | production

                                                                                                        paypal.Button.render({

                                                                                                            env: '<%=((String) session.getAttribute("MODE")).equals("LIVE") ? "production" : "sandbox"%>',

                                                                                                            locale: 'en_CA',

                                                                                                                                               style: {

                                                                                                                                            label: 'pay',

                                                                                                                                            fundingicons: true, // optional

                                                                                                                                            branding: true, // optional

                                                                                                                                            size:  'large', // small | medium | large | responsive

                                                                                                                                            shape: 'rect',   // pill | rect

                                                                                                                                            color: 'blue'   // gold | blue | silver | black

                                                                                                                                        },

                                                                                                            // Show the buyer a 'Pay Now' button in the checkout flow

                                                                                                            commit: true,

                                                                                                            // payment() is called when the button is clicked

                                                                                                            payment: function() {

                                                                                                                // Set up a url on your server to create the payment

                                                                                                                var CREATE_URL = '<%=request.getContextPath()%>/createAgreementToken.action';

                                                                                                                // Make a call to your server to set up the payment

                                                                                                                return paypal.request.post(CREATE_URL).then(function(response1) {

                                                                                                                                //alert("My TokenId: " + response1.token_id);

                                                                                                                             return response1.token_id;

                                                                                                                });

                                                                                                            },

                                                                                                            onCancel: function(data, actions) {

                                                                                                                return actions.redirect();

                                                                                                            },

                                                                                    onError: function(err) {

                                                                                                                return actions.redirect();

                                                                                                            },

                                                                                                            // onAuthorize() is called when the buyer approves the payment

                                                                                                            onAuthorize: function(data, actions) {

                                                                                                                // Set up a url on your server to execute the payment

                                                                                                                var EXECUTE_URL = '<%=request.getContextPath()%>/createAgreement.action';

                                                                                                                // Set up the data you need to pass to your server

                                                                                                                var data = {

                                                                                                                    paymentID: data.paymentID,

                                                                                                                    billingToken: data.billingToken

                                                                                                                };

                                                                                                                // Make a call to your server to execute the payment

                                                                                                                return paypal.request.post(EXECUTE_URL, data)

                                                                                                                    .then(function (res) {

console.log(res);

                                                                                                                              // return actions.redirect();

                                                                                                                              });

                                                                                                               }

                                                                                               

                                                                                                                }, '#paypal-button-container');

                                                                                                    </script>                                                                                                                         

                                                                                                                <!-- -------------------------- PayPal Button ------------- End -->

**Comment by bluepnume** [comment]
Sorry, can you please open a new issue and answer all the questions? Otherwise it's extremely difficult for us to triage.
